### ISSUE TITLE
Erronous Mag Dec Causing VRC Video Map & TwrTrainer Heading Issues

### ISSUE THREAD

**Issue by ksnifte** [vZMA/ZMA#44]
The runway centerline for Rwy 9 in KMIA EAST OPS video map doesn't match the runway.  Need to investigate the issue and apply an appropriate fix.  Not sure if the video map is wrong or if the ASDE and Runway Centerline diagrams are wrong, but I suspect the video map is wrong.

https://prnt.sc/qz8tpt

Note:  The other ops' video maps are probably also problematic.

**Comment by ksnifte** [comment]
Someone (probably Eric) mentioned, in regard to erroneous sweatbox headings that the (magnetic something) of the airport might be messed up.  Upon further inspection of the video map, it appears this is not only the case, but also is likely the cause for this ticket.

https://prnt.sc/r2vsci

**Comment by ksnifte** [comment]
Finally figured out what is going on.  There are multiple issues that were compounding together.  Also, ironically, the bug that this ticket was for is actually just a simple editing error and has nothing to do with TwrTrainer or Magnetic Declination or anything else.

## TL;DR;
Magnetic Declination in both the sector file and TwrTrainer files are all over the place, causing weird behavior in both TwrTrainer and VRC.  I propose changing the Mag Dec to +6.283° across the board (in both TwrTrainer and VRC), which will roughly best minimize these issues.  The issues cannot be perfectly fixed due to the regional nature of Mag Dec.


## Full Explanation
### **THE ISSUE:**
Pursuant to this ticket, the runway centerlines in the VRC video maps don't match the runway centerlines at KMIA.  The video maps are rotated a number of degrees.  Additionally, TwrTrainer headings at KMIA are about 13° off (e.g.: `fh 090` results in Hdg 103ish).  These issues are also present at other airports (e.g.: KTPA), but the issues aren't as bad.

### **THE CAUSE:**
The Mag Dec in the sector file is listed as `5.480`.  Since Mag Dec varies by location, the only location the sector file's Mag Dec is accurate is in the Gulf of Mexico[[1](https://maps.ngdc.noaa.gov/viewers/historical_declination/)].  VRC uses the Mag Dec to rotate the video maps so that Hdg 360 is always exactly 12:00 of the screen.  This is why the video maps don't match VRC's built-in runway centerlines-- the video maps are getting visually rotated at a different rate than the built-in centerlines.

For TwrTrainer, there are three issues at play.  
(1) The **sector's** Mag Dec at KMIA is almost 1.5° off, since `5.480` is out in the Gulf.
(2) Each airport's APT file used that specific airport's Mag Dec.  This would be fine, except none of the airports have the Mag Dec listed in the sector file (since, well, you know, it's out in the Gulf).  This introduced another ±1.5° of error in TwrTrainer headings.
(3) Ironically, I'm pretty sure there's a bug in VRC's Mag Dec interpretation.  Our APT files are "correct" by having a negative Mag Dec.  However, VRC (erroneously, IMO) expects a positive Mag Dec value in both the sector file and the APT files.  This means that our negative values in the APT files were putting all TwrTrainer headings....you guessed it.... _about 13° off_  (`6.8 + |-6.8| = 13.6`).  I checked out ZOA's sector files to confirm my suspicion, and it was confirmed.

### **PROPOSED SOLUTION**
Here is my proposed solution:

1) Establish `(+)6.283` as the official ZMA Mag Dec until it is deemed necessary to update.  This is the Mag Dec of the LBV VOR[[2](https://www.ngdc.noaa.gov/geomag/calculators/magcalc.shtml)] and is almost exactly halfway between KTPA (`5.7°`) and KMIA (`6.8`).  With an annual rate of change of 0.1°, this will serve us well for at least a few years.  [[_"Why choose the halfway point?"_: We can only choose one Mag Dec for the entire sector file (in the `[INFO]` section at the beginning).  Choosing the halfway point will make no major airports perfect, but will ensure that all major airports are reasonably and equally accurate.  The error is ±0.51°, which I would consider acceptable.]]

2) Edit the sector file and all APT files to reflect this new official Mag Dec.  **This will correct the video map/centerline issues to ±0.5°, and will completely correct all TwrTrainer issues**.

**Regression Concerns:**
- This will horribly break most of our TwrTrainer scenarios.  Our scenarios were built around this 13° error, so aircraft that are pre-defined on headings will be about 13° off.  I will go through and rebuild the scenarios that I have, but staff using personally-built scenarios will have broken scenarios.  I would propose that we start a central ZMA scenario repository so the "foundational" scenarios can be maintained and kept up to date, so staff always have access to _some_ good scenarios, even if their personally-built ones require personal updates.

- This isn't a regression concern, but rather a statement of technical debt-- the video maps at KMIA are a tiny bit sloppy, so even after correcting these issues, there will still be a slight conflict between the video maps and the built-in centerlines.  There's nothing that can be done about this without going in and perfecting the video maps (which seems unnecessary IMO).

----

Thoughts on the proposed solution? @erikquinn @FriedrichKayak
